sql/pr/sith-lords.sql

```diff
@@ -0,0 +1,45 @@
+-- Always two, there are. No more. No less. A master and an apprentice
+WITH mergers_per_year AS (
+    SELECT
+        year(c.commit_time AT TIME ZONE 'UTC') AS year,
+        ai.name as ai_name,
+        regexp_replace(ai.email, '(?<=.)[^@](?=[^@]*?@)|(?:(?<=@.)|(?!^)\G(?=[^@]*$)).(?=.*\.)', '*') AS ai_email,
+        ma.org as ai_org,
+        ci.name as ci_name,
+        regexp_replace(ci.email, '(?<=.)[^@](?=[^@]*?@)|(?:(?<=@.)|(?!^)\G(?=[^@]*$)).(?=.*\.)', '*') AS ci_email,
+        mc.org as ci_org,
+        count(*) AS commits_count
+    FROM git.default.commits c
+        JOIN memory.default.gh_idents ai ON ai.email = c.author_email OR CONTAINS(ai.extra_emails, c.author_email)
+        JOIN memory.default.gh_idents ci ON ci.email = c.committer_email OR CONTAINS(ci.extra_emails, c.committer_email)
+        LEFT JOIN members ma ON CONTAINS(ai.logins, ma.login)
+        LEFT JOIN members mc ON CONTAINS(ci.logins, mc.login)
+    WHERE ai.email != ci.email
+    GROUP BY 1, 2, 3, 4, 5, 6, 7
+),
+ranked_per_year AS (
+    SELECT
+      year,
+      ai_name,
+      ai_email,
+      ai_org,
+      ci_name,
+      ci_email,
+      ci_org,
+      commits_count,
+      row_number() OVER (PARTITION BY year ORDER BY commits_count DESC) AS in_year_rank
```

Comment:
I'm not sure it makes sense to group this by year, let's see total numbers first.

Reply:
I wanted to see if these pairings change when someone changes teams in SEP.

Reply:
This is only about Trino...